### PR TITLE
Desktop_Portfolio

### PR DESCRIPTION
**Pull request by AhmadiGu**
I built it by using HTML and CSS. it's responsive I use separate files for different sizes and import them in the style file. I make the responsiveness of it by media screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+             feature or website Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, 
+             don’t hestiate to contact me.</h3>
+             </div> 
+                <img src="images/headline.png" alt="">
+                <span class="img"></span>
+               
+                
+             
+         </section>
```
Comment (mckent05) [inline index.html]: - This section of your page doesn't look exactly like the one in the Figma template. I noticed the background image isn't displaying, please try to fix this issue. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
body hear student

Figma template:
figma2 template first page

File: index.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+                       <h3>Data Dashboard <br>
+                        Healthcare</h3>
+                       <p>A daily selection of privately personalized reads;
+                        no accounts or sign-ups required. 
+                       has been the industry's standard</p>
+                       <div class="language   container"> 
+                           <span class="html">html</span> 
+                           <span class="bootstrap boot1">bootstrap</span>
+                           <span class="ruby ">Ruby</span>
+                           </div>
```
Comment (mckent05) [inline index.html:148]: - In your mobile and desktop versions, the project cards in your works section are hardly visible. Please look into fixing this issue 🙏 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your project cards currently look like this:
project card

- In your mobile version, the content in your first project cards are overflowing, Please can you look into fixing this issue 🙏 🙏 

Your project:
first project card

File: index.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+                <div class="list-content">
+                <img class="list-icon"   src="images/Rectangle.png" alt="">
+                <span class="lang">Language</span>
+                <ul class="container">
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>HTML</li>
+                  <li>Css</li>
+                </ul>
+              </div>
```
Comment (mckent05) [inline index.html]: - The about me section of your page doesnt match the one in the Figma template, there is a missing background image in this section, please look into fixing this issue. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
about me student

Figma template:
figma template about me

File: index.html [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+           <span class="fa fa-bars"   aria-hidden="true"></span>
+         </div>
+       <ul class="menu"  >
+           <li><a class="link" href="#">protofolio</a></li>
+           <li><a class="link" href="#">about</a> </li>
+           <li><a class="link" href="#">contact</a></li>
+           <li><a class="link " href="#"><i class="fa fa-envelope-o"></i></a></li>
+       </ul>
+        
+       </nav>
```
Comment (mckent05) [inline index.html]: - In your mobile version, your navbar doesn't exactly match as it is in the Figma template. There should be equal margin both on the ;left and right. Please look into fixing this issue 👍 👍,remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project( Please pay attention to the margin on the right side of the menu bar):
navbar student

Figma template:
nabvar figma 2

File: index.html [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+          <input type="text" class="item item1" name="firstName"  placeholder="First Name" required   maxlength="30">
+          <input type="text" class="item item2" name="lastName" placeholder="Last Name" required   maxlength="30">
+            <input type="email" class="item" placeholder="Email address" name="email" required >
+           
+            <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Hello Mariane. Glad to see you |"   maxlength="500" required></textarea>
+           
+           
+          <button type="submit" class="touch">Get in touch</button>
+          
+         </form>
```
Comment (mckent05) [inline index.html:237]: - The contact form in your mobile version doesn't exactly match what is in the figma template. Please look into fixing this issue 🙏

Your project( Play pay attention to the text and the alignment of the submit button):
contact student

Figma template:
contact figma

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @gulalaiahmadi,

# STATUS: CHANGES REQUIRED ♻️ ♻️ 
 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Linters are passing correctly ✔️ ✔️ 
- GitHub flow was correctly followed 👏 


## Required Changes ♻️
- I noticed all images in your project aren't displaying correctly, you do not seem to have an `images` dir in your repo, this could be a likely cause. Please try to fix this issues and include all required images 🙏 🙏 
- There is a huge scroll on the X axis of your mobile version in your website, please look into fixing this issue so as to improve the visibility and UI of your portfolio. 👍 

Your project (Mobile version):
scroll

_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:148]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 5 above)
Comment (Abenezer-Tilahun) [reply index.html:237]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @AhmadiGu   ,
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**